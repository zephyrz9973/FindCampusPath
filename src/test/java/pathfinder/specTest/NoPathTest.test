#Test the behavior when no path found

CreateGraph graph2

AddNode graph2 A
AddNode graph2 B
AddNode graph2 C

AddEdge graph2 A B 5.0
AddEdge graph2 A C 4.0

FindPath graph2 B C
