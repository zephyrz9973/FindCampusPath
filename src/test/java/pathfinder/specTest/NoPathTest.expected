#Test the behavior when no path found

created graph graph2

added node A to graph2
added node B to graph2
added node C to graph2

added edge 5.000 from A to B in graph2
added edge 4.000 from A to C in graph2

path from B to C:
no path found
